mod_reforged/hooks/skills/actives/fire_mortar_skill.nut

```diff
@@ -0,0 +1,21 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/fire_mortar_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = "";
+	}
+
+	// Vanilla doesn't have a getTooltip function defined for this skill
+	q.getTooltip = @() function()
+	{
+		local ret = this.skill.getTooltip();
```

Comment:
This will not display AP and Fatigue cost. Use `getDefaultUtilityTooltip` instead.